presto-orc/src/main/java/com/facebook/presto/orc/reader/LongDictionaryBatchStreamReader.java

```diff
@@ -81,7 +82,7 @@ public LongDictionaryBatchStreamReader(Type type, StreamDescriptor streamDescrip
             throws OrcCorruptionException
     {
         requireNonNull(type, "type is null");
-        verifyStreamType(streamDescriptor, type, t -> t instanceof BigintType || t instanceof IntegerType || t instanceof SmallintType || t instanceof DateType);
+        verifyStreamType(streamDescriptor, type, t -> t instanceof BigintType || t instanceof IntegerType || t instanceof SmallintType || t instanceof DateType || t instanceof LongEnumType);
```

Comment:
How confident are we that the `LongDictionaryBatchStreamReader` and the `LongDirectBatchStreamReader` are the only classes that require this modification?

Reply:
I grepped for this pattern but I am not confident. What tests could I run to be more confident?